src/constants/accessories.js

```diff
@@ -737,6 +737,21 @@ export const accessories = {
     rarity: "rare",
     texture: "/head/9de3a407d2be387b45bebdd52d3d4d75a6a1b479e2b8c03267d021d4af31e072",
   },
+  SHENS_REGALIA: {
+    name: "Shen's Regalia",
+    rarity: "legendary",
+    texture: "/head/9c7fb6f414ff0eb6bf5797acb3b3af354ca3af95ceb8f3418125f6b805419f39"
+  },
+  PANDORAS_BOX: {
+    name: "Pandora's Box",
+    rarity: "common",
+    texture: "/head/fe246925cbd4c05279b9c8dcf7c2fdfc9baf9b1424aa35e501533ebb98e00522",
+  },
```

Comment:
For Pandora's Box we still need to figure out how to handle the fact that this is the only accessory that can't be recombobulated (so it shouldn't count in the total of accessories recombobulated)